README.md

```diff
@@ -151,6 +151,16 @@ After running `npm install`:
     // Use OktaSignIn
     var signIn = new OktaSignIn(/* configOptions */);
     ```
+3. [Browserify](http://browserify.org/) only: When you want to bundle your application with Browserify, you need to tell it that it doesn't have to parse the okta-signin-widget module. To do this, you need to add this flag when you run the `browserify` command:
```

Comment:
I think we can simplify this a bit:

**Note:** If you use [Browserify](http://browserify.org/) to bundle your app, you'll need to use the `--noparse` option: 

```bash
browserify main.js \
  --noparse=$PWD/node_modules/@okta/okta-signin-widget/dist/js-okta-sign-in.entry.js \
  --outfile=bundle.js
```